components/resizable/resizable.directive.ts

```diff
@@ -46,21 +47,17 @@ export interface NzResizeEvent {
   standalone: true
 })
 export class NzResizableDirective implements AfterViewInit, OnDestroy {
-  static ngAcceptInputType_nzLockAspectRatio: BooleanInput;
-  static ngAcceptInputType_nzPreview: BooleanInput;
-  static ngAcceptInputType_nzDisabled: BooleanInput;
-
   @Input() nzBounds: 'window' | 'parent' | ElementRef<HTMLElement> = 'parent';
-  @Input() nzMaxHeight?: number;
-  @Input() nzMaxWidth?: number;
-  @Input() nzMinHeight: number = 40;
-  @Input() nzMinWidth: number = 40;
-  @Input() nzGridColumnCount: number = -1;
-  @Input() nzMaxColumn: number = -1;
-  @Input() nzMinColumn: number = -1;
-  @Input() @InputBoolean() nzLockAspectRatio: boolean = false;
-  @Input() @InputBoolean() nzPreview: boolean = false;
-  @Input() @InputBoolean() nzDisabled: boolean = false;
+  @Input({ transform: numberAttributeWithZeroFallback }) nzMaxHeight?: number;
+  @Input({ transform: numberAttributeWithZeroFallback }) nzMaxWidth?: number;
```

Comment:
Since both nzMinHeight and nzMinWidth are greater than 0, I don't think it's a good idea that the fallback value of nzMax* is `0`, WDYT?